data/templates/dnsmasq/plain/resolv.dnsmasq.conf

```diff
@@ -0,0 +1,27 @@
+# According to dnsmasq's doc, dnsmasq will automatically 
+# choose the fastest-to-answer resolver as primary 
+# (if strict-order is disabled)
+
+# List taken from
+# http://diyisp.org/dokuwiki/doku.php?id=technical:dnsresolver
+
+
+# (FR) LDN
+nameserver 80.67.188.188
+# (FR) ARN
+nameserver 89.234.141.66
+# (FR) gozmail / grifon
+nameserver 89.234.186.18
+# (DE) FoeBud / Digital Courage
+nameserver 85.214.20.141
+# (DE) CCC Berlin
+nameserver 213.73.91.35
+# (DE) Ideal-Hosting
+nameserver 84.200.69.80
+nameserver 84.200.70.40
+# (DK) censurfridns
+nameserver 91.239.100.100
+nameserver 89.233.43.71
+# (FR) FDN
+nameserver 80.67.169.12
+nameserver 80.67.169.40
```

Comment:
@sachaz Thanks, I added it to the list (because it sounds legit, anyone feel free to speak up if you're against it)

Any reason why you're not on [this page](http://diyisp.org/dokuwiki/doku.php?id=technical:dnsresolver) yet ? :P